Cargo.lock

```diff
@@ -844,7 +844,7 @@ dependencies = [
  "async-trait",
  "deno_core",
  "tokio",
- "uuid 1.1.2",
+ "uuid",
```

Comment:
Was a specific version specified here for a reason? `1.1.2` might be required for compatibility somewhere.

Reply:
The dependency on uuid 0.8.2 was removed. There's a PR for that as well https://github.com/phylum-dev/cli/pull/549